oar/cli/cmd_image_signed_check.py

```diff
@@ -6,41 +6,79 @@
 from bs4 import BeautifulSoup
 from oar.core.const import *
 from oar.core.worksheet_mgr import WorksheetManager
-
+import time
 
 logger = logging.getLogger(__name__)
 
+
+# 1. go to main URL and grab all links that matches the build type
+# 2. construct a complete URL,
+def get_image_digest(url, current_try=0):
+    print(f"Getting digest from {url}")
+    res = requests.get(url)
+    if not res.ok:
+        logger.info(f"CODE: {res.status_code} -- {res.reason}")
+        raise SystemExit()
```

Comment:
suggest to raise http error via `res.raise_for_status`. `SystemExit()` will exit the interpreter directly, we don't have a chance to do error handling. e.g. mark the task as failed etc.

Reply:
done